### DISCUSSION TITLE
PolyVertexes method for Spline Entities

### DISCUSSION THREAD

**Discussion by JorenPelfrene** [DomCR/ACadSharp#478] [Ideas]
Hi, I find the PolyVertexes method very helpful for Arc and Ellipse entities. It would be great to also have this available for Spline (in the future).
It's a handy feature when using ACadSharp together with the Clipper2 library for performing boolean operations on 2D geometry.

Thank you for your great work ;)

**Comment by DomCR** [comment]
Hi @JorenPelfrene,

I'll add a feature issue for this one.

Thanks for the suggestion!
